pom.xml

```diff
@@ -0,0 +1,22 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
This pom.xml builds iris, student, and student library.  I'll add the other projects as well.